content/cybersecurity/concepts/malware/terms/keylogger/keylogger.md

```diff
@@ -0,0 +1,56 @@
+>*Keystroke logging is one of the methods to obtain sensitive informations like usernames, PINs, password and online activities from endpoint devices. An attacker can collect valuable data without entering into a strong database or file server using this method.*
+
+## How Keyloggers Work
+
+Key loggers, also known as keystroke loggers, record the keys hit on adevice and save them to a file, which is then accessed by the person who deployed the malware.
+The main purpose of keyloggers is to tamper with the chain of events that occur when a key is pressed, and information is displayed on the screen as a result of the keystroke.
+Keyloggers can be used for both lawful and illegitimate objectives,depending on the user who is utilising it.
+***Lawful Application include:***
+
+> - *Developers and analysts studying user       interaction with systems*.
+> - *Employee monitoring.*
+> - *Law enforcement or private investigators looking for evidence of an ongoing crime or inappropriate behavior.*
+> - *Quality assurance testers analyzing sources of system errors .*
+
+On the other side of the line between lawful and unlawful use, cybercriminals use keylogging technology to capture identities, confidential intellectual property, passwords and any other marketable information.
+
+### Types of Keyloggers
+
+***Keyloggers*** fall into four categories: software, hardware, wireless intercept, and acoustic.Although they differ in how they are implemented and how information is captured,these keyloggers have one thing in common.They store capture information in a log file.
+>"*When software or hardware keyloggers are used, the log files are stored on the compromised machine.
+Remote capture technologies (i.e., wireless intercept and acoustic) typically store keystroke data on the collection device*".
+
+#### 1.*Software Keyloggers*
+
+Software keyloggers capture keystroke information as it passes between the computer keyboard interface and the OS. They are implemented as traditional applications or kernel-based. In almost all malicious instances of this type of keylogger, users participated in some way in the software’s installation.
+
+#### 2.*Hardware Keyloggers*
+
+A hardware keylogger is essentially a circuit located somewhere between the keyboard and the computer. The keylogger is connected directly to the PC and the keyboard to the keylogger.
+Another method is to install a keylogger circuit into a standard keyboard.This has the advantage of no physical evidence of user monitoring.
+
+#### 3.*wireless intercept Keyloggers*
+
+A wireless intercept keylogger usees wireless Technologies based on the capabality of transmission.For instance Bluetooth-accessible keylogger.
+The one big disadvantage of using wireless intercept keyloggers is the need for a receiver/antenna relatively close to the target system because the wireless devices are limited range of transmission.
```

Comment:
```suggestion
The one big disadvantage of using wireless intercept keyloggers is the need for a receiver/antenna relatively close to the target system because the wireless devices have a limited range of transmission.
```